index.bs

```diff
@@ -139,6 +165,8 @@ Every {{DocumentOrShadowRoot}} has <dfn>adopted stylesheets</dfn>.
 The user agent must include all style sheets in the {{DocumentOrShadowRoot}}'s
 [=adopted stylesheets=] inside its <a>document or shadow root CSS style sheets</a>. The [=adopted stylesheets=] are ordered after all the other style sheets (i.e. those derived from {{DocumentOrShadowRoot/styleSheets}}).
 
+If a {{DocumentOrShadowRoot}} is moved into a different {{Document}}, its [=adopted stylesheets=] are also moved along with it, retaining the same set of style sheets.
```

Comment:
This statement is unclear to me. Is this a statement of fact that follows from another part of the spec, and is trying to summarize things in an easy to understand way? If so, I am having a hard time understanding it. Or is this trying to impose a normative implementation requirement? If so, I'm unsure what parts of the code would need to be modified to make this true.

Reply:
Hmm yeah I'm not sure about this too, it's to resolve #46. Do you have a better place and wording for it?

Reply:
Well, how are you planning on implementing this?

Reply:
On the implementation side, it's just automatically carried over along with the `DocumentOrShadowRoot` since it's the same object, and the `CSSStyleSheet` objects are not changed in any way, the base URLs stay the same, the @imports are not re-resolved, etc. 
 
Maybe we should modify "adopt a node" spec as well?

Reply:
I see. I think it's automatic in the spec too. I would turn it into a non-normative note, and rephrase it to be clearer with something like

```html
<p class="note">
Because the [=adopted stylesheets=] are a property of the {{DocumentOrShadowRoot}}, they move along with the {{ShadowRoot}} if it gets [=adopted=] into a different {{Document}}, e.g. when adopting its [=shadow host=].
</p>
```

I guess I'm still not sure what @bzbarsky was concerned about in #46 though, so maybe he can chime in.

Reply:
Changed that into a note, does this cover your concern well @bzbarsky?